CHANGES.md

```diff
@@ -10,7 +10,10 @@
 - Add partial support for the match statement. As it's experimental, it's only enabled
   when `--target-version py310` is explicitly specified (#2586)
 - Add support for parenthesized with (#2586)
-- Fixed Vim with Python 3.10 by removing deprecated distutils (#2610)
+
+### Integrations
+
+- Fixed vim plugin with Python 3.10 by removing deprecated distutils import (#2610)
 - Declare support for Python 3.10 for running Black (#2562)
```

Comment:
This ended up in the wrong section, can you move it up?

Reply:
But vim plugin is integration issue, isn't it?

Reply:
I'm referring to the Python 3.10 line

Reply:
I see, I've moved it.